minikube/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+# configmap.yaml
+kind: ConfigMap 
+apiVersion: v1 
+metadata:
+  name: module-14
+data:
+  FLASK_APP: "app"
+  FLASK_ENV: "development"
+  LOG_LEVEL: "DEBUG"
+  OAUTHLIB_INSECURE_TRANSPORT: "1"
+  LOGIN_DISABLED: "TRUE"
```

Comment:
Minor, but worth noting you check for `True` rather than `TRUE` so this isn't actually being applied